app/queries/draft_and_live_versions.rb

```diff
@@ -0,0 +1,56 @@
+module Queries
+  module DraftAndLiveVersions
+    extend ArelHelpers
+    def self.call(content_item, target_class, locale = nil, base_path = nil)
```

Comment:
I think the variable naming of target_class is a little misleading and would be better represented by either:
- Change variable name to target_table_name which more closely represents how it's used
- pass in the model class itself and then run `table_name` on it to get the table name